### DISCUSSION TITLE
Impianto di domotica cablata

### DISCUSSION THREAD

**Discussion by massimob** [indomus/forum#729] [Hardware & Firmware]
Salve a tutti,
sto ristrutturando i bagni di casa e vorrei procedere con il rendere "smart" gli attuatori nelle stanze, ovviamente è un bagno e non sono presenti più di un paio di punti luce e una tapparella, ma vorrei in ogni caso procedere per preparare il terreno per la ristrutturazione del resto.
Premetto che sono circa un paio di anni che utilizzo Home Assistant con svariate integrazioni, non mi considero un esperto ma una soluzione si trova sempre.

Sono tutti molto belli gli interruttori/dispositivi Wi-Fi o a radio frequenze già presenti e utilizzati, ma per svariati motivi, che non elenco per non annoiarvi, vorrei utilizzare un impianto cablato.
Dalle mie ricerche l'unico degno di nota è il By-me della Vimar che sembra abbia una discreta vita da essere stabile e il filo utilizzato per il bus può essere affiancato alla normale linea elettrica.
Vorrei trovare una soluzione che utilizzi per sensori, interruttori e attuatori un bus cablato da connettere poi all'home assistant. Non ho problemi in questa fase a fare due serie di canaline separate, una per il bus e una per la linea elettrica.

Intanto grazie per la risposta e complimenti per il sito e per tutti i dettagliatissimi articoli

**Comment by tempod** [comment]
Ciao.
Io ho un sistema vimar byme (non il plus uscito recentemente). Per mia esperienza personale ti posso dire che è affidabilissimo. E' ormai up da circa 4 anni e non da mai problemi. Fa quello che deve sempre. Io ho portato tutto su home assistant perchè anche se il sistema della vimar ha un sacco di componenti non è ovviamente integrabile con tutta la miriade di componenti che esiste. Purtroppo però il nuovo sistema plus, da quel che so, non è integrabile con home assistant. Ma, il 99% dei componenti del plus sono gli stessi del vecchio, quindi al limite potresti usare il vecchio web server.

**Reply by massimob** [reply]
ok grazie mille indago, ma non ci sono altre soluzioni di domotica cablata?

**Reply by tempod** [reply]
In che senso?
Le più famose sono ovviamente vimar e bticino ma, per esempio,  esistono anche soluzione di gewiss, ave, e tanti altri.
Per l'integrazione in HA però ne ho la certezza solo per quello della vimar e dalla bticino.

**Reply by massimob** [reply]
Sono settimane che cerco ma non trovavo nulla con ricerche generiche, grazie mille

Intanto provo a dare un occhiata alla bticino, visto che ho tutto bticino.

**Reply by tempod** [reply]
informati bene sulla bticino perchè quello integrabile in home assistant mi pare di aver capito che deve passare per homekit
